virtual_ecosystem/models/litter/litter_model.py

```diff
@@ -189,6 +189,7 @@ def update(self, time_index: int, **kwargs: Any) -> None:
 
         Args:
             time_index: The index representing the current time step in the data object.
+            kwargs: Further arguments to the update method.
```

Comment:
I think I've had situations where putting `kwargs` and not `**kwargs` in a docstring has broken things, but that may have been with `mkdocs` rather than `sphinx`.